testing/base/source_gen.h

```diff
@@ -173,7 +241,9 @@ class SourceGen {
   // as `GetShuffledUniqueIdentifiers`.
   //
   // Usually, benchmarks should use the shuffled version. However, this is
-  // useful when there is already a post-processing step to shuffle things.
+  // useful when the identifiers will be used in multiple different ways and
+  // only then shuffled, or if there is already a post-processing step to
+  // shuffle things which is very expensive in debug builds.
```

Comment:
```suggestion
```suggestion
  // useful when deterministic access to the identifiers is needed to avoid
  // introducing noise, or if there is already a post-processing step to shuffle
  // things, since shuffling is very expensive in debug builds.
``````